core/Cargo.toml

```diff
@@ -69,6 +69,7 @@ indexmap = "2.2.6"
 
 [dev-dependencies]
 iroha_executor_data_model = { workspace = true }
+iroha_wasm_builder = {workspace = true }
```

Comment:
nit (formatting)

```suggestion
iroha_wasm_builder = { workspace = true }
```

Reply:
removed since `iroha_wasm_builder` is no longer required